### DISCUSSION TITLE
This is RIDICULOUS

### DISCUSSION THREAD

**Discussion by nonogrub** [Chia-Network/chia-blockchain#4647] [General]
In a month we will need 10Tb per day of Plots to maintain our "estimated time to win" without changes....
![Captura de pantalla 2021-05-12 a las 8 29 20](https://user-images.githubusercontent.com/83576173/117929208-cf0a6600-b2fc-11eb-8190-6caaf2ea4295.png)

**Comment by ilyakooo0** [comment]
At some point the farmers will catch up to HDD production rates and the growth will slow down (:

**Reply by nonogrub** [reply]
hope so
